lib/cog_api/http/relay_groups.ex

```diff
@@ -38,27 +38,23 @@ defmodule CogApi.HTTP.RelayGroups do
     |> ApiResponse.format_delete("The relay group could not be deleted")
   end
 
-  def add_relay(%{name: name}, %{relay: relay_name}, %Endpoint{}=endpoint) do
-    {relay, relay_group} = get_group_relay(name, relay_name, endpoint)
-    update_membership(relay_group.id, relay.id, :add, endpoint)
-  end
-  def add_relay(relay_group_id, relay_id, %Endpoint{}=endpoint) do
-    update_membership(relay_group_id, relay_id, :add, endpoint)
-  end
-
-  def remove_relay(%{name: name}, %{relay: relay_name}, %Endpoint{}=endpoint) do
-    {relay, relay_group} = get_group_relay(name, relay_name, endpoint)
-    update_membership(relay_group.id, relay.id, :remove, endpoint)
+  def update_memberships(action, %{name: name}, %{relay: relay_name}, %Endpoint{}=endpoint) do
```

Comment:
Can we make `update_memberships` take a list of relays? That will correspond to the pattern in `update_assignments`. The api already supports it.